native/run

```diff
@@ -28,7 +28,7 @@ program
       .replaceAll('\n', ' ')
```

Comment:
I get an error while executing yarn android:
PS C:\Projects\Integreat\integreat-app\native> yarn android
yarn run v1.22.10
$ yarn android:integreat-test-cms
$ node ./run android integreat-test-cms
C:\Projects\Integreat\integreat-app\native\run:28
      .replaceAll('\n', ' ')
       ^

TypeError: execSync(...).toString(...).replaceAll is not a function
    at Command.<anonymous> (C:\Projects\Integreat\integreat-app\native\run:28:8)
    at Command.listener [as _actionHandler] (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:488:17)
    at C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1227:65
    at Command._chainOrCall (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1144:12)
    at Command._parseCommand (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1227:27)
    at Command._dispatchSubcommand (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1050:25)
    at Command._parseCommand (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:1193:19)
    at Command.parse (C:\Projects\Integreat\integreat-app\native\node_modules\commander\lib\command.js:897:10)
    at Object.<anonymous> (C:\Projects\Integreat\integreat-app\native\run:34:9)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
error Command failed with exit code 1.

Reply:
@Cmd190 Can you try replacing it with `.replace(/\n/g, ' ')`?

Reply:
What is your node version?

Reply:
@klinzo Node Version v14.17.3

Reply:
> @Cmd190 Can you try replacing it with `.replace(/\n/g, ' ')`?

The error is still the same

Reply:
You mean it is now `....replace(...)` is not a function instead of `....replaceAll(..)`?

Reply:
I oversaw the change from `replaceAll` to `replace`  and just changed the Regex. After using only `replace` is build correctly

Reply:
Same goes for yarn start

Reply:
Perfect, if you want you can commit your change here and start your branch from there.